templates/app/models/sequelize-model.hbs

```diff
@@ -23,17 +23,25 @@ module.exports = (sequelize, DataTypes) => {
   // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
   {{modelName}}.associate = (models) => {
     {{#each references as |reference|}}
-    {{../modelName}}.belongsTo(models.{{reference.ref}}, {
+    {{#if reference.isBelongsToMany}}
+    {{../modelName}}.belongsToMany(models.{{reference.ref}}, {
```

Comment:
You could factorize here:
```suggestion
    {{../modelName}}.{{reference.association}}(models.{{reference.ref}}, {
```
and use the `{{#if reference.isBelongsToMany}}`  for the association options that differs between `belongsToMany` and `hasOne`/`hasMany`.